app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -70,7 +70,7 @@ import org.simple.clinic.util.UuidRoomTypeConverter
     InstantRoomTypeConverter::class,
```

Comment:
I'd had this discussion with Saket earlier about unifying the type adapters since they are functionally the same, but keeping them separate for consistency makes sense for now. Let's have a discussion about whether we should unify them across the codebase later.